src/main/resources/db/changelog/changelog.sql

```diff
@@ -208,3 +208,29 @@ CREATE TABLE IF NOT EXISTS `RequestInfos` (
 
 ALTER TABLE `AccountLanguages`
 ADD COLUMN `order_index` int(8) DEFAULT 0;
+
+-- changeset bridge:7
+
+CREATE TABLE IF NOT EXISTS `FileMetadata` (
+    `studyId` varchar(255) NOT NULL,
+    `guid` varchar(60) NOT NULL,
+    `name` varchar(255) DEFAULT NULL,
+    `createdOn` BIGINT UNSIGNED NOT NULL,
+    `modifiedOn` BIGINT UNSIGNED NOT NULL,
+    `description` text,
+    `mimeType` varchar(255) DEFAULT NULL,
+    `deleted` tinyint(1) NOT NULL DEFAULT '0',
+    `version` int(10) unsigned NOT NULL,
+    PRIMARY KEY (`guid`),
+    KEY `Studies_idx` (`studyId`)
```

Comment:
Do we also want a compound index by studyId and guid?

Reply:
Yes, we ensure study correctness by using both so we should add an index.